### PR TITLE
Add Option to Hide Banner

### PR DESCRIPTION
**Pull request by Dan-Bird**
This PR amends the compass banner used within the blog template to now include a switch that can hide the banner.

### CODE REVIEW

File: version_control/Codurance_September2020/modules/Blog Post Banner.module/fields.json [hunk 1]
```diff
@@ -1,135 +1,157 @@
...
+    "label": "Hide Banner?",
+    "required": false,
+    "locked": false,
+    "type": "boolean",
+    "default": false
+  },
+  {
+    "id": "82bf4763-110a-df04-3585-5698c89e4e41",
+    "name": "backround_image",
+    "label": "Backround Image",
```
Comment (przemek-codurance) [inline version_control/Codurance_September2020/modules/Blog Post Banner.module/fields.json:13]: backround -> background ?

Comment (Dan-Bird) [reply version_control/Codurance_September2020/modules/Blog Post Banner.module/fields.json]: Nice spot. I fixed the typo here: 68c14e7e55b20ae8c5e22e4cd26af254f4cf5d2a

File: version_control/Codurance_September2020/modules/Blog Post Banner.module/module.html [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {% if module.button_type == "std" or module.button_type == "cta" or module.content or module.backround_image.src  %}
-<div class="in-page-banner page-center in-page-banner--even container section custom-pt-200 custom-pb-100">
+<div
+  class="in-page-banner page-center in-page-banner--even container section custom-pt-200 custom-pb-100"
+  {% if module.hide_banner %}
```
Comment (przemek-codurance) [inline version_control/Codurance_September2020/modules/Blog Post Banner.module/module.html:4]: hide_banner is some kind of variable inside the module?

Comment (Dan-Bird) [reply version_control/Codurance_September2020/modules/Blog Post Banner.module/module.html:4]: Anything from `module` would come from the fields you have set up. So within the `fields.json` file there is an entry with the `name: hide_banner` which creates a switch within the UI when editing the blog template on Hubspot.

Comment (przemek-codurance) [reply version_control/Codurance_September2020/modules/Blog Post Banner.module/module.html:4]: thanks!

**Review by przemek-codurance — APPROVED** [review]

**Pull request merged**